### PR TITLE
17718: Setting 'PROVIDER_DB_DRIVER' env-var to the upper case value of 'dataType' env-var"

### PR DESCRIPTION
**Pull request by victoralfaro-dotcms**

### CODE REVIEW

File: images/dotcms/ROOT/srv/entrypoint.sh [hunk 1]
```diff
@@ -23,15 +23,25 @@ if [[ "${1}" == "dotcms" || -z "${1}" ]]; then
 
     cd /srv/home
 
+    if [ ! -z "${WAIT_DB_FOR}" ]; then
```
Comment (victoralfaro-dotcms) [inline images/dotcms/ROOT/srv/entrypoint.sh:26]: This too. Adding some extra time to wait for the database to be fully usable.
In Oracle it takes a while.

File: images/dotcms/ROOT/srv/50-database-config.sh [hunk 2]
```diff
@@ -6,7 +6,11 @@ source /srv/utils/config-defaults.sh
 
 
 echo "Database config ...."
-[[ -z "${PROVIDER_DB_DNSNAME}" ]] && PROVIDER_DB_DRIVER="POSTGRES"
+if [[ -z "${PROVIDER_DB_DNSNAME}" ]]; then
```
Comment (victoralfaro-dotcms) [inline images/dotcms/ROOT/srv/50-database-config.sh]: This is the real change. The rest are victims of my IDE's auto formatter.

**Review by dsilvam — APPROVED** [review]

File: images/dotcms/ROOT/srv/50-database-config.sh [hunk 3]
```diff
@@ -6,7 +6,11 @@ source /srv/utils/config-defaults.sh
...
+  PROVIDER_DB_DRIVER="H2"
```
Comment (nollymar) [inline images/dotcms/ROOT/srv/50-database-config.sh]: the default database for testing is postgres in case the env var is not provided

Comment (victoralfaro-dotcms) [reply images/dotcms/ROOT/srv/50-database-config.sh]: Fixed

**Review by nollymar — APPROVED** [review]

**Pull request merged**